service/dynamodb/dynamodbbatchwriter/batchwriter.go

```diff
@@ -0,0 +1,129 @@
+package dynamodbbatchwriter
```

Comment:
I think we want to move away from having the service name in the higher abstraction packages. Let's rename this to something like `batchwriter`.